telegram/ext/updater.py

```
@@ -112,7 +112,7 @@ def _thread_wrapper(self, target, *args, **kwargs):
     def start_polling(self,
                       poll_interval=0.0,
                       timeout=10,
-                      network_delay=2,
+                      total_timeout=15,
```

@leandrotoledo i'm not so sure. i prefer failures rather than silent errors